metrics/web/lib/common/presentation/routes/metrics_page_route.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+class MetricsPageRoute<T> extends MaterialPageRoute<T> {
+  MetricsPageRoute({
+    @required WidgetBuilder builder,
+    RouteSettings settings,
+  }) : super(
```

Comment:
It seems like this class should accept all parameters available in the `MaterialPageRoute`. Could you please add missing parameters here?